src/sentry/issues/utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, Mapping
+
+from sentry.issues.grouptype import PerformanceNPlusOneGroupType
+from sentry.issues.issue_occurrence import IssueOccurrence
+from sentry.models import Project
+from sentry.utils.performance_issues.performance_problem import PerformanceProblem
+
+
+def can_create_group(
+    ocurrence: IssueOccurrence, options: Mapping[str, Any], project: Project
```

Comment:
Why do we pass options in here instead of importing?

Reply:
oh I thought passing in a dependency explicitly is better but yeah fixed it to use the import instead